dpxdt/tools/site_diff.py

```diff
@@ -228,6 +228,12 @@ def run(self,
         good_urls = set()
 
         yield heartbeat('Scanning for content')
+        
+        http_username=None
```

Comment:
Style nit, can you add spaces around the assignment?

http_username = None
http_password = None